src/views/chart.slm

```diff
@@ -0,0 +1,31 @@
+- pattern = 'Chart';
+
+/ Layout
+= extend('layouts/default');
+
+/ Variables
+= partial('partials/styles.slm');
+= partial('partials/links.slm');
+
+/ Partials
+= partial('partials/head.mixin.slm');
+= partial('partials/header.mixin.slm');
+= partial('partials/nav.mixin.slm');
+= partial('partials/section.mixin.slm');
+= partial('partials/section-full.mixin.slm');
+
+/ Content blocks
+= content('head');
+  = mixin('head', pattern);
+
+= content('header');
+  = mixin('header');
+
+= content('content');
+  h1 class='${class_demo_headers}' = pattern;
+
+  = mixin('section', 'objects/chart/chart');
```

Comment:
This mixin pulls the `objects/chart/chart.md` file and the `objects/chart/chart.slm` in to the page.